test/tools/ossfuzz/README.md

```diff
@@ -1,8 +1,63 @@
 ## Intro
 
-[oss-fuzz][1] is Google's fuzzing infrastructure that performs continuous fuzzing. What this means is that, each and every upstream commit is automatically fetched by the infrastructure and fuzzed.
+[oss-fuzz][1] is Google's fuzzing infrastructure that performs continuous fuzzing. What this means is that, each and every upstream commit is automatically fetched by the infrastructure and fuzzed on a daily basis.
 
-## What does this directory contain?
+## How to build fuzzers?
+
+We have multiple fuzzers, some based on string input and others on protobuf input. To build them, please do the following:
+
+- Create a local docker image from `Dockerfile.ubuntu1604.clang.ossfuzz` in the `.circleci/docker` sub-directory. Please note that this step is going to take at least an hour to complete. Therefore, it is recommended to do it when you are away from computer (and the computer is plugged to power since we do not want a battery drain).
+
+```
+$ cd .circleci/docker
+$ docker build -t solidity-ossfuzz-local -f Dockerfile.ubuntu1604.clang.ossfuzz .
+```
+
+- Create and login into the docker container sourced from the image built in the previous step from the solidity parent directory
```

Comment:
Is `Create` a typo here? `Create and login into` sounds a bit weird to me

Reply:
Perhaps we should just say "Login to the docker container..."?